docs/src/api_comparison.md

```diff
@@ -88,9 +88,9 @@ welcome.
 | **Segmentation**                      |                                           |                                   |                                           |
 | Connected components                  | `label_components`                        | `label`                           | `bwconncomp`, `bwlabel`                   |
 | Foreground/background                 |                                           | `active_contour`                  | `activecontour`                           |
-| Clustering                            |                                           | `quickshift`                      |                                           |
+| Clustering                            | ` kmeans`, `fuzzy_cmeans`                 | `quickshift`                      |                                           |
```

Comment:
Maybe we should add `mean_shift` to the Julia column, `slic` to the python column.

Reply:
@timholy  under Clustering section of segmentation or else where??

Reply:
Yeah, that's a clustering-based algorithm.